languages/rust/exercises/concept/tuples/.meta/example.rs

```diff
@@ -0,0 +1,66 @@
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub enum Unit {
+    Millilitre,
+    UsCup,
+    UsFluidOnce,
+    UsTeaspoon,
+    UsTablespoon,
+}
+
+pub fn get_volume(volume: &(Unit, f32)) -> f32 {
+    volume.1
+}
+
+pub fn to_cup(volume: &(Unit, f32)) -> (Unit, f32) {
+    match volume {
+        (Unit::Millilitre, volume)  => {
+            let converted = volume * (1.0/240.0);
+            (Unit::UsCup, format!("{:.2}", converted).parse::<f32>().unwrap())
```

Comment:
For cases where you really do want to truncate to two decimal digits, `(100.0 * n).round() / 100.0` is a much faster, better approach than converting to a string using the format machinery. There's absolutely nothing in the documentation or problem statement indicating why you'd want to truncate this at all, though.

Reply:
> Why on earth are you truncating to two decimal digits by means of string conversions?

Because I am still learning Rust and was feeling expressly lazy. 😄